skada/_subspace.py

```diff
@@ -297,13 +299,11 @@ def adapt(self, X, y=None, sample_domain=None, **kwargs):
         -------
         X_t : array-like, shape (n_samples, n_components)
             The data transformed to the target subspace.
-        y_t : array-like, shape (n_samples,)
-            The labels (same as y).
         sample_domain : array-like, shape (n_samples,)
             The domain labels transformed to the target subspace
             (same as sample_domain).
-        weights : None
-            No weights are returned here.
+        weigts : None
+            No weigts are returned here.
```

Comment:
seems like typos came back :)

Reply:
yes, you are right, I'm correcting that right now